HardcodeTray/enum.py

```diff
@@ -74,6 +74,7 @@ class ApplicationType:
     NWJS = "NWJSApplication"
     QT = "QtApplication"
     APPLICATION = "Application"
+    JS = "JavaScriptApplication"
```

Comment:
I would use B64ElectronApplication or something similar.

Reply:
Oops, copy-paste. ;)

Reply:
I agree, this looks clearer.